setup.py

```diff
@@ -17,19 +17,15 @@ def find_stub_files(name: str) -> List[str]:
     return result
 
 
-with open("README.md", "r") as f:
+with open("README.md") as f:
     readme = f.read()
 
-dependencies = [
-    "mypy>=0.782,<0.790",
-    "typing-extensions",
-    "django",
-]
+dependencies = ["mypy>=0.782", "typing-extensions>=3.7.2", "django>=2.2.0"]
```

Comment:
`mypy>=0.790`

Reply:
are you sure we want to restrict like this? I'd say 0.750 and above, we dont want to restrict people